lib/membrane_element_portaudio/sink.ex

```diff
@@ -6,12 +6,20 @@ defmodule Membrane.Element.PortAudio.Sink do
   alias Membrane.Buffer
   alias Membrane.Caps.Audio.Raw, as: Caps
   alias Membrane.Element.PortAudio.SyncExecutor
+  alias Membrane.Time
   alias __MODULE__.Native
+
   import Mockery.Macro
+
```

Comment:
If you want, change order of directives. Up for your judgement as well as in the other case.
https://github.com/christopheradams/elixir_style_guide#modules